docker/datahub-gms/env/docker.env

```diff
@@ -17,6 +17,7 @@ GRAPH_SERVICE_IMPL=neo4j
 ENTITY_REGISTRY_CONFIG_PATH=/datahub/datahub-gms/resources/entity-registry.yml
 MAE_CONSUMER_ENABLED=true
 MCE_CONSUMER_ENABLED=true
+PE_CONSUMER_ENABLED=true
```

Comment:
do we need to add these changes to mae-consumer's docker.env as well?

Reply:
you also need to run the generate quickstart script

Reply:
Oh good call - yes!